synapseclient/client.py

```diff
@@ -2619,7 +2619,7 @@ def setPermissions(
         Arguments:
             entity: An Entity or Synapse ID to modify
             principalId: Identifier of a user or group. '273948' is for all registered Synapse users
-                            and '273949' is for public access.
+                            and '273949' is for public access. None implies public access.
```

Comment:
Does `None` imply both public and all registered Synapse users OR just public access?

Reply:
I copied the comment over from one of the methods this calls. But after setting it to None and running the code it sets `Anyone on the web` to those permissions:
![image](https://github.com/Sage-Bionetworks/synapsePythonClient/assets/17128019/9dc515af-4566-4d78-93e8-232f6ffbbded)

"Only registered synapse users" permissions are not touched